agent/bdiagent.py

```diff
@@ -38,32 +38,32 @@ def getpercepts(self, beliefbase):
 
     def manage_goals(self, beliefbase, goalbase):
         for goal in self.goalbase:
-            if (self.is_achieved(goal)):
+            if self.is_achieved(goal):
                 print(goal, " Goal Achieved!")
                 self.achieved_goal(goal, goalbase)
         return
 
     # A goal is achieved if either it shares a name with a belief and the belief is true or its associated function returns true
     def is_achieved(self, goal):
-        if (goal in self.beliefbase.keys()):
-            if (self.beliefbase[goal] == 1):
+        if goal in self.beliefbase.keys():
+            if self.beliefbase[goal] == 1:
                 return 1
             return 0
         else:
             goalfunc = self.goal_functions[goal]
-            if (goalfunc()):
+            if goalfunc():
                 return 1
             return 0
 
     # When a goal is achieved it is removed from the goalbase, if it is a subgoal of some other goal then the next subgoal should now be attempted
     def achieved_goal(self, goal, goalbase):
         goalbase.remove(goal)
-        if (goal in self.pending_goals.keys()):
+        if goal in self.pending_goals.keys():
             next_goallist = self.pending_goals[goal]
             next_goal = next_goallist.pop(0)
             self.add_goal(next_goal)
             self.pending_goals.pop(goal, None)
-            if (len(next_goallist) > 0):
+            if len(next_goallist) > 0:
```

Comment:
May just be personal preference but you can get rid of the > 0 here since if it's 0 it will be false anyway

Reply:
Some regex showed this happened a lot cheers for pointing it out, I'll adjust the respective lines